spec/views/hq/users/new.html.erb_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper' 
+
+describe 'hq/users/new.html.erb', type: :view do
+
+  before :each do
+    assign :user, build_stubbed(:user)
```

Comment:
If you use `User.new` here it shouldn't have an `:id`, more similar to the object that would come from the controller